library/src/main/java/com/novoda/downloadmanager/lib/DownloadService.java

```diff
@@ -419,9 +419,12 @@ private void deleteFileAndMediaReference(DownloadInfo info) {
 
     private boolean kickOffDownloadTaskIfReady(boolean isActive, DownloadInfo info) {
```

Comment:
No, `isActive` lives outside of the loop so it'll be set to true after it iterates over an active download